src/title/TitlePrivate.js

```diff
@@ -50,5 +51,5 @@ class TitlePrivate extends PureComponent<Props> {
 
 export default connect((state, props) => ({
   user: getUserInPmNarrow(props.narrow)(state),
-  presence: getPresence(state),
+  presence: getPresenceByEmail(state, props.narrow[0].operand),
```

Comment:
As I mentioned earlier, it would be a nice followup to refactor this part away. I think the point should be that `TitlePrivate` is always for a PM narrow -- there's always exactly one other user it's about -- and so instead of `narrow` it should take a prop `email`.

That may require some improved abstractions in `utils/narrow`, for `Title` to handle that. Hmm.

Reply:
OK, made some new abstractions! See `git log --stat -p --reverse 4e7062eb4..e9fe1e801`.

I've demonstrated by converting `TitlePrivate` itself. I'd be glad to see a followup PR to do the same thing for the other `Title*` components, as well as the other places that use the same pattern -- `git grep isFunc`.